### PR TITLE
Better logging for swarm tests

### PR DESCRIPTION
**Pull request by cpuguy83**
Call helper for starting swarm agents and add some logging with daemon id's when joining the swarm.

### CODE REVIEW

File: internal/test/daemon/swarm.go [hunk 1]
```diff
@@ -47,13 +47,17 @@ func (d *Daemon) StartAndSwarmInit(t testingT) {
...
+		th.Helper()
+	}
 	d.StartNode(t)
 
 	tokens := leader.JoinTokens(t)
 	token := tokens.Worker
 	if manager {
 		token = tokens.Manager
 	}
+	t.Logf("[%s] Joining swarm manager [%s]@%s, swarm listen addr %s", d.id, leader.id, leader.SwarmListenAddr(), d.SwarmListenAddr())
```
Comment (thaJeztah) [inline internal/test/daemon/swarm.go]: Lowercase `J` perhaps (to be consistent with the other log)

```suggestion
	t.Logf("[%s] joining swarm manager [%s]@%s, swarm listen addr %s", d.id, leader.id, leader.SwarmListenAddr(), d.SwarmListenAddr())
```

**Review by vdemeester — APPROVED** [review]
LGTM with @thaJeztah suggestion :hugs:

**Review by kolyshkin — APPROVED** [review]
LGTM

**Comment by cpuguy83** [comment]
All green and has @thaJeztah's change.
Merging.

**Pull request merged**